manifests/init.pp

```diff
@@ -75,6 +74,7 @@
   Boolean $manage_config                 = true,
   Boolean $manage_install                = true,
   Boolean $manage_dependencies           = true,
+  Array[String] $vcs_dependencies        = [],
```

Comment:
I think allowing empty strings doesn't make sense here so we can prohibit it?
```suggestion
  Array[String[1]] $vcs_dependencies     = [],
```

Reply:
Updated